test/local/statsd_tests.js

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var tap = require('tap')
+var test = tap.test
```

Comment:
> You might like to use our local promisified ptaptest wrapper from test/ptaptest.js for consistency, although it won't provide you any value here since you're not using promises.

Yeah I saw it but it was not useful